keps/0010-package-manager.md

```diff
@@ -166,12 +167,82 @@ Our official repository is hosted on Google Cloud Storage and following a flat s
 
 We rely on just an HTTP Server, e.g. the out-of-the-box solution that Google Cloud Storage provides, that serves operator `tgz` files and makes them available to users.
 
-The logic for keeping the operators in sync should live in the CLI and is not defined on this KEP . That way the HTTP server only has to serve the index and the Package `tgz` files, without having to implement any business logic. 
+The logic for keeping the operators in sync should live in the CLI and is not defined on this KEP . That way the HTTP server only has to serve the index and the Package `tgz` files, without having to implement any business logic.
 
 The proposed structure is fairly easy to replicate and highly customizable.
 
 Safety when distributing our Packages is another concern. As we continue working on this KEP we will add more details on how we can verify and prevent `Arbitrary software installation`, `Vulnerability to key compromises`, etc. For now using HTTPS to fetch the index and Packages is considered sufficient.
 
+### Index File Specification
+
+`index.yaml` is the base definition of a repository.  It follows the following reference format.
+
+```yaml
+apiVersion: str(desc='version of repository')
+entries: map[string]PackageVersions
+  str(desc='name of operator'):
+  - name:  str(max=253)
+    version: semver()
+    appVersion: str(desc='component controlled version', required=False)
+    home: url(desc='home page, git repo', required=False)
```

Comment:
in operator we call this url, I think home is a bit too generic here. I was immediately thinking about home directory :D I would personally omit this value from index altogether, it's inside operator, we can always add it here if the metadata is needed. It's hard to remove, but it's easy to add :)

Reply:
this is already part of our code... are you suggesting that we remove it from code? or just not expose it?

Reply:
I'm not sure who was initially responsible for the original code and yaml... this yaml has urls, home and source.  There is a url in operator we should consolidate on the same nomenclature.  

I'll look into that.  Also agree on easier to add in later.